index.html

```diff
@@ -33,5 +33,155 @@ <h1>
+        </div>
+        <div class="card">
+          <div class="card_container">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <div class="tags">
+              <ul>
+                <li><span> Ruby on rails</span></li>
+                <li><span>css</span></li>
+                <li><span>JavScript</span></li>
+                <li><span>html</span></li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card_container">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <div class="tags">
+              <ul>
+                <li><span> Ruby on rails</span></li>
+                <li><span>css</span></li>
+                <li><span>JavScript</span></li>
+                <li><span>html</span></li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card_container">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <div class="tags">
+              <ul>
+                <li><span> Ruby on rails</span></li>
+                <li><span>css</span></li>
+                <li><span>JavScript</span></li>
+                <li><span>html</span></li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="aboutme">
+      <div class="about_desc">
+        <h2>About me</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button>Get My Resume</button>
+      </div>
+      <div class="card_about">
+        <div class="cardabt">
+          <div class="imageabout">
+            <img src="images/icons/about3.png" alt="" />
+          </div>
+          <h3>Languages</h3>
+          <div class="tagsabout">
+            <ul>
+              <li><span>JavaScript</span></li>
+              <li><span>Ruby</span></li>
+              <li><span>Html </span></li>
+              <li><span>CSS</span></li>
+            </ul>
+          </div>
+        </div>
+        <div class="cardabt">
+          <div class="imageabout">
+            <img src="images/icons/about3.png" alt="" />
+          </div>
+          <h3>Frameworks</h3>
+          <div class="tagsabout">
+            <ul>
+              <li><span>Bootstrap</span></li>
+              <li><span>Ruby on rails</span></li>
+              <li><span>RSpec </span></li>
+              <li><span>Capybara</span></li>
+              <li><span>Selenium</span></li>
+            </ul>
+          </div>
+        </div>
+        <div class="cardabt">
+          <div class="imageabout">
+            <img src="images/icons/about3.png" alt="" />
```

Comment:
[OPTIONAL] Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image